testdata/source/ofx/test_fidelity_ira/journal.beancount

```diff
@@ -0,0 +1,139 @@
+option "operating_currency" "USD"
+option "conversion_currency" "USD"
+
+1900-01-01 open Assets:Investment:Fidelity
+  ofx_org: "fidelity.com"
+  ofx_broker_id: "fidelity.com"
+  account_id: "313379972"
+  ofx_account_type: "securities_and_cash"
+  div_income_account: "Income:Fidelity:Dividends"
+  interest_income_account: "Income:Fidelity:Interest"
+  capital_gains_account: "Income:Fidelity:Capital-Gains"
+  fees_account: "Expenses:Investment:Fidelity:Fees"
+  commission_account: "Expenses:Investment:Fidelity:Commissions"
+
+
+1900-01-01 open Assets:Investment:Fidelity:Cash
+1900-01-01 open Assets:Investment:Fidelity:AAPL220624P120
+1900-01-01 open Assets:Investment:Fidelity:DAL220520C42
+1900-01-01 open Assets:Investment:Fidelity:IWM220826C207
+1900-01-01 open Assets:Investment:Fidelity:MSFT220826C310
+1900-01-01 open Assets:Investment:Fidelity:WMT220617P118
+1900-01-01 open Expenses:Investment:Fidelity:Fees
+1900-01-01 open Expenses:Investment:Fidelity:Commissions
+1900-01-01 open Expenses:FIXME
+1900-01-01 open Income:Fidelity:Capital-Gains
+1900-01-01 open Income:Fidelity:Capital-Gains:DAL220520C42
+
+2022-05-13 * "SELLOPT - DAL220520C42"
```

Comment:
Can we add an extra keyword in the description to indicate whether it's sell to open or sell to close?
I know it's in the metadata, but it'd make it easier to understand activity at a glance. Plus the description field is pretty short at the moment, so no downside.